app/src/main/res/values/strings.xml

```diff
@@ -207,73 +209,73 @@
     <string name="pref_name_tts_previousButtonIsRewind">\"Previous\" button rewinds</string>
-    <string name="pref_desc_tts_previousButtonIsRewind">Pressing \"previous\" media button rewinds instead of skipping to previous article</string>
-    <string name="pref_name_tts_showPreviewAsAlbumArt">Use preview as album art</string>
-    <string name="pref_desc_tts_showPreviewAsAlbumArt">Use the article preview image (if it is present) as album art (visible in notification and may be displayed as the lockscreen background)</string>
-
-    <string name="pref_categoryName_sync">Synchronization</string>
-    <string name="pref_name_sync_syncTypes">Synchronization types</string>
-    <string name="pref_desc_sync_syncTypes">Press to show more info about synchronization types</string>
-    <string name="pref_desc_sync_syncTypes_text">There are two main synchronization types: \"Full\" and \"Fast\".\n\n
-        \"Full\" sync deletes all local content and downloads it again. It is available from the options menu, as a last resort.\n\n
-        \"Fast\" sync (incremental) only downloads changes made since the previous sync, reducing time and bandwidth spent. However, this method has a shortcoming: It can\'t detect remotely deleted article (so no local articles are deleted during \"Fast\" sync). Pull downwards beyond the top position scrollbar in any article list to initiate.\n\n
-        To remedy the shortcoming of the \"Fast\" sync type, \"Deleted article sweep\" checks every local article and asks the server whether it is still present on the server - if not, the local article is removed. Depending on the number of articles you have, it takes a while, but saves traffic over the \"Full\" sync.\n\n
-        None of these functions affect client-to-server sync: Your local changes will always be sent to the server before updating local content.
+    <string name="pref_desc_tts_previousButtonIsRewind">Pressing \"Previous\" media button rewinds instead of skipping to previous article</string>
+    <string name="pref_name_tts_showPreviewAsAlbumArt">Preview image as album art</string>
+    <string name="pref_desc_tts_showPreviewAsAlbumArt">Use the article preview picture (if present) as the cover for albums (shown in notification possibly as lockscreen background)</string>
+
+    <string name="pref_categoryName_sync">Syncing</string>
+    <string name="pref_name_sync_syncTypes">Ways to sync</string>
+    <string name="pref_desc_sync_syncTypes">Press for more info about syncing types</string>
+    <string name="pref_desc_sync_syncTypes_text">\"Full\" and \"Fast\" are two main syncing types.\n\n
+        \"Full\" deletes all local content and re-downloads it. (Always in the options menu as a last resort.)\n\n
+        \"Fast\" (incremental) downloads any changes made since the previous sync to reduce time and bandwidth. Remotely deleted articles are not detected or deleted. (Pull ↓ beyond the top position scrollbar in any article list to use).\n\n
```

Comment:
```suggestion
        \"Fast\" (incremental) downloads any changes made since the previous sync to reduce time and bandwidth. It can\'t detect or delete remotely deleted articles. (Pull ↓ beyond the top position scrollbar in any article list to use it).\n\n
```